build.sbt

```diff
@@ -138,8 +174,11 @@ lazy val generic = (project in file("generic"))
     libraryDependencies ++=
       COMPILE(
         "com.chuusai" %% "shapeless" % shapelessVersion
-      )
+      ),
+    Test / scalacOptions += "-Wconf:cat=other-pure-statement&msg=org.scalatest.Assertion:s"
   )
+  .dependsOn(`test-helpers` % "test->test")
+  .settings(scalacOptions += "-Wconf:origin=neotypes.generic.implicits.given:s")
```

Comment:
This is necessary to use this hack https://github.com/neotypes/neotypes/pull/667/files#r1305977657

Reply:
No longer necessary.